Firmware/communication/can/can_simple.cpp

```diff
@@ -343,6 +346,24 @@ bool CANSimple::get_vbus_voltage_callback(const Axis& axis) {
     return canbus_->send_message(txmsg);
 }
 
+bool CANSimple::get_adc_voltage_callback(const Axis& axis, const can_Message_t& msg) {
+    can_Message_t txmsg;
+
+    txmsg.id = axis.config_.can.node_id << NUM_CMD_ID_BITS;
+    txmsg.id += MSG_SEND_ADC_VOLTAGE;
```

Comment:
Here I am

Reply:
Ah this was changed in a later commit so I didn't see it. Yeah this should be `MSG_GET_ADC_VOLTAGE`.